e2e/network-switch.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { expect, test } from '@playwright/test';
```

Comment:
In general, I don't really like creating an E2E test for this. It's pretty complex fo what it's testing and it's not obvious from the implementation what the E2E test is testing. The way I see it, the fix about making sure that react query caches the query for block txs correctly by incorporating the network key in the query key. We don't want to test react query and whether it correctly caches the query based on the query key provided, so the only thing to test is that the query key is included in the query provided to the use query hook, and we know it is because it is a required argument to the function for generating the query key, so I don't think a test is really necessary here